pkg/blobaccess/forward.go

```diff
@@ -0,0 +1,103 @@
+package blobaccess
+
+import (
+	"io"
+
+	"github.com/mandelsoft/vfs/pkg/vfs"
+	"github.com/opencontainers/go-digest"
+
+	"github.com/open-component-model/ocm/pkg/blobaccess/bpi"
+	"github.com/open-component-model/ocm/pkg/blobaccess/file"
+	"github.com/open-component-model/ocm/pkg/blobaccess/standard"
+)
+
+///////////
+// Standard
+///////////
+
+func DataAccessForReaderFunction(reader func() (io.ReadCloser, error), origin string) bpi.DataAccess {
+	return standard.DataAccessForReaderFunction(reader, origin)
+}
+
+// DataAccessForBytes wraps a bytes slice into a DataAccess.
+// Deprecated: used DataAccessForData.
+func DataAccessForBytes(data []byte, origin ...string) bpi.DataSource {
+	return standard.DataAccessForData(data, origin...)
+}
+
+// DataAccessForData wraps a bytes slice into a DataAccess.
+func DataAccessForData(data []byte, origin ...string) bpi.DataSource {
+	return standard.DataAccessForData(data, origin...)
+}
+
+func DataAccessForString(data string, origin ...string) bpi.DataSource {
+	return standard.DataAccessForString(data, origin...)
+}
+
+// ForString wraps a string into a BlobAccess, which does not need a close.
+func ForString(mime string, data string) BlobAccess {
+	return standard.ForString(mime, data)
+}
+
+func ProviderForString(mime, data string) BlobAccessProvider {
```

Comment:
```suggestion
func BlobAccessProviderForString(mime, data string) BlobAccessProvider {
```

Reply:
Same here `blobaccess.ProviderForXXX(...)`.

Reply:
similar to dataaccess this would lead to a bunch of additional packages. Therefore (as before) , there is only the `blobaccess`package. It contains
- the `DataAccess` base abstraction (plus the creator functions `DataAccesForXXX(...)
- the `BlobAccess`  abstraction extending the `DataAccess  (plus the creator functions `ForXXX(...)
- the `BlobAccessProvider` abstraction  (plus the creator functions `ProviderForXXX(...)